### PR TITLE
second contribution

### PR DESCRIPTION
**Pull request by deepthi1107**

### CODE REVIEW

**Comment by deepthi1107** [comment]
@MridulS, review my PR.

File: 2022-round-1/deepthi1107/nx_tutorial_script.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+# In[3]:
+
+
+#adding node 11,"N1",1,2,3
+graph1.add_node(11)
+graph1.add_node("N1")
+L=[1,2,3]
+graph1.add_nodes_from(L)
```
Comment (MridulS) [inline 2022-round-1/deepthi1107/nx_tutorial_script.py:25]: Thanks @deepthi1107!

Could you also add a `tuple` object as a node. Currently there are only `int` and `str` type objects.

Comment (deepthi1107) [reply 2022-round-1/deepthi1107/nx_tutorial_script.py:25]: sure, @MridulS.

**Comment by deepthi1107** [comment]
Hello @MridulS, I have made changes.

**Comment by MridulS** [comment]
Thanks @deepthi1107!

**Pull request merged**